exec/java-exec/src/main/resources/rest/index.ftl

```diff
@@ -17,17 +17,56 @@
   <a href="/queries">back</a><br/>
   <div class="page-header">
   </div>
-  <div class="table-responsive">
-    <table class="table table-hover">
-      <tbody>
-        <#list model as stat>
-          <tr>
-            <td style="border:none;"><b>${stat.getName()}</b></td>
-            <td style="border:none; font-family: Courier;">${stat.getValue()}</td>
-          </tr>
-        </#list>
-      </tbody>
-    </table>
+
+  <#list model.getDrillbits() as drillbit>
+    <#if !drillbit.isVersionMatch()>
+      <div id="message" class="alert alert-danger alert-dismissable">
```

Comment:
This seems to emit a warning about version mismatch for every mismatched Drillbit. Seems it would be better to list the Drillbits, and look for mismatches. Then, if found, perhaps:

Highlight the mismatched version (in red or whatever) or each Drillbit, then, after the list of bits, display the message that "One or more drill bits, highlighted above, have versions different from the cluster version of x.y.z."